README.md

```diff
@@ -57,6 +57,10 @@ let result = await nango.auth('github', '<user-id>');
 let token = await nango.getToken('github', '<user-id>');
 ```
 
+Or you can also choose to deploy directly to render<br/>
```

Comment:
It would be great to move this to the Quickstart section, creating 2 sub-headers: 1. 1-click deploy 2. Run locally. Similarly to https://github.com/lightdash/lightdash

Reply:
Done.